suite/suite_order_test.go

```diff
@@ -0,0 +1,47 @@
+package suite
```

Comment:
What is your logic for putting this in a separate file? Does it not make sense to just have this as another test in `suite_test.go`?

Reply:
The other file becoming huge, thought could separate the tests based on what we integration vs small func tests.

Reply:
I see. I do agree that it's becoming a bit large but I don't think navigating it is still a huge problem. 

I'd be happy to discuss any ideas you might have about improving its readability & maintainability, whether it involves splitting it up or some other way.